lib/integrations/externalauditstorage/bootstrap.go

```diff
@@ -243,19 +283,32 @@ func createAthenaWorkgroup(ctx context.Context, clt BootstrapAthenaClient, workg
 // * CreateDatabase
 // * CreateTable
 // * UpdateTable
-func createGlueInfra(ctx context.Context, clt BootstrapGlueClient, table, database, eventBucket string) error {
+func createGlueInfra(ctx context.Context, clt BootstrapGlueClient, accountID, region, table, database, eventBucket string, ownershipTags map[string]string) error {
 	fmt.Printf("Creating Glue database %s\n", database)
 	_, err := clt.CreateDatabase(ctx, &glue.CreateDatabaseInput{
 		DatabaseInput: &gluetypes.DatabaseInput{
 			Name:        &database,
 			Description: aws.String(glueDatabaseDescription),
 		},
 	})
-	if err != nil {
-		var aee *gluetypes.AlreadyExistsException
-		if !errors.As(err, &aee) {
-			return trace.Wrap(err, "creating Glue database")
+
+	var aee *gluetypes.AlreadyExistsException
+	switch {
+	case err == nil:
+		// Tag created resource.
+		// Only Glue databases can be tagged. Glue tables do not have tags.
+		// https://docs.aws.amazon.com/glue/latest/dg/monitor-tags.html
+		databaseARN := glueDatabaseARN(region, accountID, database)
+		if _, err := clt.TagResource(ctx, &glue.TagResourceInput{
+			ResourceArn: aws.String(databaseARN),
+			TagsToAdd:   ownershipTags,
+		}); err != nil {
+			return trace.Wrap(err, "adding tags %+v to Glue Database %q", ownershipTags, databaseARN)
```

Comment:
nit: I don't think %+v does anything special for map[string]string
```suggestion
			return trace.Wrap(err, "adding tags %v to Glue Database %q", ownershipTags, databaseARN)
```

Reply:
We ended up adding the tags during creation 👍